apps/src/studio/studio.js

```diff
@@ -153,6 +154,17 @@ function loadLevel() {
   // protagonistSpriteIndex was originally mispelled. accept either spelling.
   Studio.protagonistSpriteIndex = level.protagonistSpriteIndex || level.protaganistSpriteIndex;
 
+  var customGameType = level.customGameType || '';
+  switch (customGameType.replace(/ /g,'').toLowerCase()) {
```

Comment:
Do you think it would be better if I was more prescriptive rather than allowing whitespace/case difference? I suppose I could do that and have a dropdown instead of free text on LB.